includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -2130,4 +2131,46 @@ public static function filter_bulk_post_updated_messages( $messages, $bulk_count
 
 		return $messages;
 	}
+
+	/**
+	 * Is AMP Enabled on Post
+	 *
+	 * @param WP_Post $post              Post object to check.
+	 * @param array   $validation_errors Validation errors.
+	 * @param array   $counts            Error counts.
+	 *
+	 * @return bool|void
+	 */
+	public static function is_amp_enabled_on_post( $post, $validation_errors = array(), $counts = array() ) {
```

Comment:
I think it would be better to eliminate the `$validation_errors` and `$counts` params. This simplifies the API. Any previously-obtained terms will already be stored in the object cache, so there is not really much concern about duplicated work.